push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/AdmPushNotificationSender.java

```diff
@@ -34,6 +35,11 @@
 public class AdmPushNotificationSender implements PushNotificationSender {
     private final Logger logger = LoggerFactory.getLogger(AdmPushNotificationSender.class);
 
+    private static final Counter promPrushRequestsAdm = Counter.build()
+            .name("aerogear_ups_push_requests_adm")
```

Comment:
What does the `adm` here stand for? Admin? Is there a way to trigger a batch requests in the UI?

Reply:
adm -> Amazon device messaging (the class is also named ADMpush sender)